src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -36,18 +34,17 @@ class DoctrineChoiceLoader extends AbstractChoiceLoader
      *
      * @param string $class The class name of the loaded objects
```

Comment:
```suggestion
     * @param class-string $class The class name of the loaded objects
```

Reply:
We use the the EM to resolve the actual class from this string which could be the ORM's legacy short notation. We could deprecate not passing an actual FQCN though, but I'd like to do that in a separate PR.

Reply:
Well, if you pass a legacy short notation, Doctrine ORM will trigger a deprecation warning in `$manager->getClassMetadata($class)` in the constructor. So we are already covered here. No need for a duplicate deprecation warning.
The code will be simplified once we drop support for ORM 2.x (which will probably be in Symfony 8.0)

Reply:
We could simplify it for 7.0 already by not resolving the class name through the EM anymore.